wolox-books/src/app/screens/auth/components/modal/modal.component.scss

```diff
@@ -0,0 +1,80 @@
+@import '../../../../../scss/colors';
+
+.modal {
+  background-color: $white;
+  border-radius: 20px;
+  display: flex;
+  flex-direction: column;
+  left: 25%;
+  max-height: 900px;
+  padding: 40px;
+  position: fixed;
+  top: 25%;
+  width: 900px;

```

Comment:
Remember that if you set a fixed width, it will not adapt properly. Try setting a 
```
max-width: 900px;
width: 100%;
```